### DISCUSSION TITLE
Nicht nur Zielmenge sondern auch Bunkermenge beim Abtanken

### DISCUSSION THREAD

**Discussion by BierNot72** [VertexDezign/TargetFillLevel#6] [Ideas]
Beim Abtanken eines Dreschers mit Ladewagen wird mir aktuell nicht angezeigt, wieviel der Drescher noch gefüllt ist. Das selbe beim Befüllen vom Lagersilo.
Wäre toll, wenn ihr das in den Mod auch noch einbauen könnt. Was meint ihr dazu?

**Comment by schliesser** [comment]
Heyho @BierNot72 

danke für dein Feedback. Die Kombination eines Dreschers mit Ladewagen hab ich definitiv nicht getestet. Aber ich habe das Abtanken eines Dreschers in ein Silo getestet, das hatte funktioniert. Der Mod sollte mit der neuen Version jetzt auch bei Produktionen und Hofsilos den Füllstand anzeigen. 🤔 

Kannst du mir ein paar Screenshots von deinen Fahrzeugkombinationen beim Abladen machen und hier anhängen?
Falls du Mods verwendest wär es super, wenn du mir die Links zum Download der Mods gibst.

Viele Grüße

**Reply by BierNot72** [reply]
Hello @schliesser
Beim Abtanken mit der Drescher / LKW-Anhängerkonfiguration wird ja vom Drescher zum Anhänger die Füllmenge im Anhänger durch euren Mod angezeigt. Fahre ich aber den LKW und lasse den Drescher mit CP laufen, wäre es toll, wenn mir den Tankinhalt vom Drescher auch angezeigt wird. Somit hat man dann alles auf einen Blick und macht dies Sache sehr übersichtlich. 

Ich nutze die folgende Fahrzeugkonfiguration für's Abtanken
Drescher: [CLAAS LEXION Modelljahr 2023](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=252639&title=fs2022)
LKW: [Roadrunner+](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=230640&title=fs2022)
Anhänger: [Kroeger Agroliner SMK34](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=285297&title=fs2022)

![Abtanken Drescher 30](https://github.com/user-attachments/assets/56d815af-7c77-44a7-a65b-db12e3b1c9bf)
![Abtanken LKW 30](https://github.com/user-attachments/assets/8debc522-d891-45f2-a1c6-ee79ed9d07d0)


und für's Füllen als Beispiel einer Sämaschine
Traktor: [Fendt 900 Vario Series](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=225936&title=fs2022)
Sämaschine: [John Deere 1775NT 2022](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=232099&title=fs2022)
Nachfülltanks: [Platzierbare Nachfülltanks](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=228672&title=fs2022)

![Füllen vom Silo 30](https://github.com/user-attachments/assets/5e2af385-7559-475c-9ed6-5f7dba0a59f7)

Genügt dir das mal so?

**Reply by schliesser** [reply]
Vielen Dank @BierNot72 

Jetzt hab ich verstanden, was du meinst. Die Idee gefällt mir gut! Muss schauen ob ich auch anders herum vom Zielfahrzeug aus an die Füllstände komme.

Ich notiere mir die Idee für den FS25.

**Reply by BierNot72** [reply]
Das ist doch perfekt so, dann haben wir ja noch einen Grund mehr, uns auf den neuen FS25 zu freuen 👍
